lucetc/tests/wasm.rs

```diff
@@ -78,6 +78,32 @@ mod module_data {
         assert_eq!(mdata.function_info().len(), 3);
         assert_eq!(mdata.export_functions()[0].names, vec!["main"]);
     }
+
+    #[test]
+    fn duplicate_imports() {
+        let m = load_wat_module("duplicate_imports");
+        let b = Bindings::from_file(&PathBuf::from(
+            "tests/bindings/duplicate_imports_bindings.json",
+        ))
+        .unwrap();
+        let h = HeapSettings::default();
+        let c = Compiler::new(&m, OptLevel::Fast, &b, h).expect("compile duplicate_imports");
+        let mdata = c.module_data().unwrap();
+
+        assert_eq!(mdata.import_functions().len(), 2);
+        assert_eq!(mdata.import_functions()[0].module, "env");
+        assert_eq!(mdata.import_functions()[0].name, "read");
+        assert_eq!(mdata.import_functions()[1].module, "env");
+        assert_eq!(mdata.import_functions()[1].name, "write");
+        assert_eq!(mdata.function_info().len(), 5);
+        assert_eq!(mdata.function_info()[0].name, Some("host_read"));
+        assert_eq!(mdata.function_info()[1].name, Some("host_write"));
+        assert_eq!(mdata.function_info()[2].name, Some("guest_func__start"));
+        assert_eq!(mdata.export_functions().len(), 1);
```

Comment:
This test brushes up with exported imports not being fully supported yet (see #203) - this test will change once that PR is in as there should be three exported functions ("read", "write", and "_start")

Reply:
So it looks like this test fail once #203 lands? Excellent